js/ui/themes.js

```diff
@@ -292,11 +300,13 @@ function detachCssClasses(element) {
 
 init({
     _autoInit: true,
-    loadCallback: function() {
-        themeReadyCallback.fire();
-    }
+    _forceTimeout: true
 });
 
+function themeReady(callback) {
+    themeReadyCallback.add(callback);
```

Comment:
Callback will never call if theme was loaded early. Is it expected behaviour?

Reply:
Yes. We need to subscribe on `themes.ready` before themes loading